app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-key-auth-plugin.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enable Key Authentication for Application Registration
+---
+
+## Overview
+
+You can use the Key Authentication plugin for authentication in conjunction with
+the Application Registration plugin.
+
+The key auth plugin uses the same Client ID as generated for the Kong OAuth2 plugin.
+You can use the same Client ID credential for a Service that has the OAuth2 plugin enabled.
+
+## Prerequisites
+
+* {{site.ee_product_name}} is installed, version 2.2.1.0 or later.
+* Create a Service.
+* Activate your application for a Service if you have not already done so. The
+Service Contract must be approved by an Admin if auto approve is not enabled.
+* Enable the [Application Registration plugin](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/enable-application-registration) on a Service.
```

Comment:
I think the order of these two bullet points should be swapped. Application Registration needs to be applied before a developer can request access to the service. 
```suggestion
* Enable the [Application Registration plugin](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/enable-application-registration) on a Service.
* Activate your application for a Service if you have not already done so. The
Service Contract must be approved by an Admin if auto approve is not enabled.
```